great-idea/css/index.css

```diff
@@ -63,4 +63,157 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	margin: 40px 80px 150px 80px;
+}
+nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+nav a {
+	text-decoration: none;
+	color: lightgray;
+}
+
+.index-header {
+	margin-top: 60px;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-bottom: 60px;
+}
+
+.index-header h1 {
+	font-size: 90px;
+}
+
+.index-header h1 {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	align-items: center;
+}
+
+.index-header img {
+	width: 500px;
+	height: 500px;
+}
+
+.index-header h1 button {
+	border: 1px solid black;
+	margin-top: 20px;
+	width: 200px;
+	height: 40px;
+	font-size: 20px;
+}
+
+.top-content {
+	display: flex;
+	justify-content: center;
+	margin-bottom: 50px;
+}
+
+.middle-img {
+	width: 100%;
+}
+
+.bottom-content {
+	display: flex;
+	justify-content: space-evenly;
+	margin-top: 50px;
+}
+
+address {
+	margin-top: 30px;
+}
+
+footer {
+	margin-top: 40px;
+	margin-bottom: 80px;
+	text-align: center;
+}
+
+.services-header {
+	margin-top: 40px;
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	justify-content: flex-start;
+}
+
+.services-header img {
+	width: 100%;
+}
+
+.services-header div {
+	margin-top: 30px;
+}
+
+.services-header h1 {
+	font-size: 35px;
+}
+
+.services-body {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
+	justify-content: space-between;
+}
+
+.services-body div{
+	width: 45%;
+	border: 1px solid black;
+	padding: 15px;
+	margin: 25px 0px;
+	background: lightgray;
+}
+
+.services-body div button {
+	border: 1px solid black;
+	margin-top: 15px;
+	width: 150px;
+	height: 40px;
+}
+
+.bottom-content div {
+	display: flex;
+	flex-direction: column;
+	width: 45%;
+}
+
+.bottom-content h3 {
+	font-size: 30px;
+}
+
+.bottom-content p {
+	margin: 30px 0px; 
+}
+
+.bottom-content ul li {
+	list-style-type: square;
+	margin: 30px 17px;
+}
+
+.bottom-content {
+	display: flex;
+	align-items: flex-start;
+	justify-content: space-between;
+}
+
+.bottom-content img {
+	width: 45%;
+}
+
+footer {
+	margin-top: 80px;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
Hmmm not working on my side, not sure I understand "only screen" --> https://css-tricks.com/css-media-queries/